src/Tvheadend.cpp

```diff
@@ -2256,10 +2344,29 @@ void CTvheadend::ParseChannelDelete(htsmsg_t* msg)
   }
   Logger::Log(LogLevel::LEVEL_DEBUG, "delete channel %u", u32);
 
-  /* Erase */
+  /* We need to check and update providers. May be the deleted channel is the last channel with this provider */
+  int32_t providerUid{0};
```

Comment:
Should this too be `PVR_PROVIDER_INVALID_UID`?

Reply:
Yes. I will fix that.

Reply:
Done.